src/Sensors.cpp

```diff
@@ -610,9 +610,12 @@ bool Sensors::CO2CM1106Init() {
     DEBUG("-->[CM1106] Serial number:", sensor.sn);
     DEBUG("-->[CM1106] Software version:", sensor.softver);
 
-    // // Setup ABC parameters
-    // DEBUG("-->[CM1106] Setting ABC parameters...");
-    // sensor_CM1106->set_ABC(CM1106_ABC_OPEN, 7, 415);    // 7 days cycle, 415 ppm for base
+    // Setup ABC parameters
+    DEBUG("-->[CM1106] Setting ABC parameters...");
+    sensor_CM1106->set_ABC(CM1106_ABC_OPEN, 7, 415);    // 7 days cycle, 415 ppm for base
```

Comment:
Some questions please:

These parameters are for specific sensor or all sensors of this type?
These parameters will be executed in each init() or setup() , are this ok? or we need run this only some times?

Reply:
It is only for the specific CM1106SL-NS.
In order to avoid mistakes, if you connect a new unit, it comes with mode A by default, and if it connected to cananirio, the meassure is not correct. It is necesary to changue this working mode to B, to work with 3,3 V uart.
It is neccesary only one time, this mode is saved in the sensor and it can be changued when you want with the command sensor_CM1106->set_working_status(1); or sensor_CM1106->set_working_status(0); 1 for B, 0 for A

Reply:
for the moment, for the sensors library, maybe we need separate this flow in a different method, maybe like a `Sensors::CO2CM1106calibrate()`? or `Sensors::CO2CM1106config()`? Because, we haven't any flow for now, from the Android app, but here, in the library for other projects, we could need it.  You can do it?